src/control_helpers/air_actions_tracking.rs

```diff
@@ -134,6 +134,48 @@ impl TnuaSimpleAirActionsCounter {
         }
     }
 
+    /// Resets the air actions counter to a specific count, excluding the current action.
```

Comment:
"excluding the current action" is misleading - it makes it sound like the `air_count_for` will not be affected for the current action - but it will.

Actually, I think the current action is going to mess this up. If you use `reset_count_to(0)` (or `reset_count()` when there is an active action, `actions_including_current` will be 0 and then when `air_count_for` calculates `self.actions_including_current - 1 ` for the current action - it'll underflow.

If we want this to work, we'll have to change the `current_action` field to `Option<(&'static str, usize)>`, and have it also store the `actions_including_current` at the time the action was registered (before the counter was increased)

Reply:
I think i did use a saturating subtraction so there is no underflow problem, but I did not think it would mess things up with the `current_action` like that. Apologies for this oversight.